protocol-units/bridge/contracts/src/AtomicBridgeInitiatorMOVE.sol

```diff
@@ -97,32 +92,35 @@ contract AtomicBridgeInitiatorMOVE is IAtomicBridgeInitiatorMOVE, OwnableUpgrade
 
     function completeBridgeTransfer(bytes32 bridgeTransferId, bytes32 preImage) external onlyOwner {
         BridgeTransfer storage bridgeTransfer = bridgeTransfers[bridgeTransferId];
-        if (bridgeTransfer.state != MessageState.INITIALIZED) revert BridgeTransferHasBeenCompleted();
-        if (keccak256(abi.encodePacked(preImage)) != bridgeTransfer.hashLock) revert InvalidSecret();
-        if (block.timestamp > bridgeTransfer.timeLock) revert TimelockExpired();
+        require(bridgeTransfer.state == MessageState.INITIALIZED, BridgeTransferHasBeenCompleted());
+        require(keccak256(abi.encodePacked(preImage)) == bridgeTransfer.hashLock, InvalidSecret());
+        require(block.timestamp <= bridgeTransfer.timeLock, TimelockExpired());
+
         bridgeTransfer.state = MessageState.COMPLETED;
 
         emit BridgeTransferCompleted(bridgeTransferId, preImage);
     }
 
     function refundBridgeTransfer(bytes32 bridgeTransferId) external onlyOwner {
         BridgeTransfer storage bridgeTransfer = bridgeTransfers[bridgeTransferId];
-        if (bridgeTransfer.state != MessageState.INITIALIZED) revert BridgeTransferStateNotInitialized();
-        if (block.timestamp < bridgeTransfer.timeLock) revert TimeLockNotExpired();
+        require(bridgeTransfer.state == MessageState.INITIALIZED, BridgeTransferStateNotInitialized());
+        require(block.timestamp >= bridgeTransfer.timeLock, TimeLockNotExpired());
+
         bridgeTransfer.state = MessageState.REFUNDED;
         
         // Decrease pool balance and transfer MOVE tokens back to the originator
         poolBalance -= bridgeTransfer.amount;
-        if (!moveToken.transfer(bridgeTransfer.originator, bridgeTransfer.amount)) revert MOVETransferFailed();
```

Comment:
This is not related to input validation, so may not be modified to use a `requires`.